app/knowledge_panels.py

```diff
@@ -39,3 +41,54 @@ def hunger_game_kp(
             ],
         },
     }
+
+
+def last_edits_kp(
+    facet: str,
+    value: Union[str, None] = None,
+    country: Union[str, None] = None,
+):
+    if facet == "packaging":
+        plural = facet
+    else:
+        plural = facet_plural(facet=facet)
+    query = {
+        "fields": "code,last_editor,last_modified_t",
+        "sort_by": "last_modified_t",
+    }
+    if facet == "country":
+        country_code = country_to_ISO_code(value=value)
+        country = value
+        url = f"https://{country_code}-en.openfoodfacts.org"
+    else:
+        if value is not None:
+            url = "https://world.openfoodfacts.org"
+            query[f"{plural}_tags_en"] = value
+        if country is not None:
+            country_code = country_to_ISO_code(value=country)
+            url = f"https://{country_code}-en.openfoodfacts.org"
+
```

Comment:
@alexgarel  result of `if country is not None` this loop isn't showing on FastAPI swagger UI but running perfect on my terminal ?